### PR TITLE
Readonly indicator.

### LINKED ISSUE #7629: Make read-only clearer.
Not to annoy user, I suggest a warning on first edit.

I got surprised using google drive where the file was read-only a few time. 
Especially since autosaved is disabled in read-only, so you don't especially notice that you might 
loose work.

### PR DESCRIPTION
**Pull request by jdfreder**
screen shot 2015-02-09 at 11 11 45 am

screen shot 2015-02-09 at 11 11 55 am

closes #7629

### CODE REVIEW

**Comment by jdfreder** [comment]
@ellisonbg design review.  @Carreau who reported the issue.

**Comment by jdfreder** [comment]
@Carreau this PR doesn't add a one time warning dialog though...  You may still want that.

**Comment by Carreau** [comment]
> this PR doesn't add a one time warning dialog though... You may still want that.

No I think the icon is visually enough. If we show a warning that should be on first modif, which would be painful to do I guess. 

+1

File: IPython/html/templates/notebook.html [hunk 1]
```diff
@@ -62,6 +62,12 @@
               <span class="kernel_indicator_name">Kernel</span>
               <i id="kernel_indicator_icon"></i>
             </p>
+            <i id="readonly-indicator" class="navbar-text">
```
Comment (Carreau) [inline IPython/html/templates/notebook.html]: `title='read-only notebook'` ?

Comment (jdfreder) [inline IPython/html/templates/notebook.html]: Yeah good point.

**Comment by jdfreder** [comment]
@Carreau tooltip added.

**Comment by minrk** [comment]
Makes sense. @jdfreder can you make the indicator start hidden, so it isn't shown while notebooks are loading?

**Comment by jdfreder** [comment]
> . @jdfreder can you make the indicator start hidden, so it isn't shown while notebooks are loading?

Yeah, I can do that.  I actually thought of that and intentionally didn't do it since you actually can't save during that period since the notebook hasn't loaded completely.  But I have objection to changing it, seeing that it could be annoying to see that icon the 99% of time that you open a notebook (since opening a notebook that is actually read-only is probably rare).

**Comment by jdfreder** [comment]
@minrk last commit hides it by default.

**Comment by minrk** [comment]
Thanks, :+1:

**Pull request merged**